lndclient.go

```diff
@@ -79,6 +79,7 @@ func (h *lndHtlcEventsClient) recvInternal() (*resolvedEvent, error) {
 			htlc:    event.OutgoingHtlcId,
 		},
 		timestamp: time.Unix(0, int64(event.TimestampNs)),
+		// Note: outgoing peer is not reported here, as it is not available.
```

Comment:
I understand that this is a quick way to do it, but it also breaks the abstraction. If not too cumbersome, maybe it works to make a parent struct that embeds resolvedEvent, adds the outgoing peer and instantiate that in eventLoop?

Reply:
Looks much better this way!

Reply:
I think so too, thanks for fixing.